index.cjs

```diff
@@ -784,6 +784,11 @@ safeql: try {
               transform: '{type}[]',
             },
           ],
+          overrides: {
+            types: {
+              json: 'JsonAgg[]',
```

Comment:
The first solution was to map `json` to `JasonAgg` as in this commit https://github.com/upleveled/eslint-config-upleveled/pull/239/commits/825a4d3fc5a7ed2136862bec362821ba6c5971b9 following [this comment](https://github.com/ts-safeql/safeql/issues/102#issuecomment-1318322571) from @Newbie012, but this expects `JsonAgg` to always be an object, but we know that the `json_agg` aggregate function always returns an array.

![Screenshot 2023-05-22 at 11 18 14](https://github.com/upleveled/eslint-config-upleveled/assets/74430629/869af1f1-76f0-46de-95e2-b79c8ba15db3)

- No error because `JsonAgg` is an object
![Screenshot 2023-05-22 at 11 18 34](https://github.com/upleveled/eslint-config-upleveled/assets/74430629/ab13c42e-73a3-44b2-aaf2-047cba15d578)


- The correct behavior should be as shown in the screenshots below, `JsonAgg` should always be an array
![Screenshot 2023-05-22 at 11 21 28](https://github.com/upleveled/eslint-config-upleveled/assets/74430629/9e960698-1764-443c-ba74-24536a53cbcd)

- Throw an error when it's not
![Screenshot 2023-05-22 at 11 21 53](https://github.com/upleveled/eslint-config-upleveled/assets/74430629/acc501fc-7d24-4a40-b119-633d361c188a)

That's why this commit was made
https://github.com/upleveled/eslint-config-upleveled/pull/239/commits/db05e4ba7b9bf62c1ff7306b705d3b351abe821c

Reply:
nice, thanks for the comment here!

maybe you can write a comment about this in the SafeQL issue and confirm that:

- `JsonAgg` is incorrect
- `JsonAgg[]` is correct

Reply:
After the conversation with Eliya, he advised that I use the suggestion he gave in the issue comment I posted on the description of this PR, He gave some reasons which is included these screenshots below
https://github.com/upleveled/eslint-config-upleveled/pull/239/commits/317c0eb9dadda3b3db02a0858a779485e843982a

<img width="1007" alt="Screenshot 2023-05-23 at 05 57 41" src="https://github.com/upleveled/eslint-config-upleveled/assets/74430629/64c1c87e-f0ea-4293-8e38-6ca02cdc3d48">

<img width="1007" alt="Screenshot 2023-05-23 at 05 58 03" src="https://github.com/upleveled/eslint-config-upleveled/assets/74430629/e8d9a19a-89b2-4edd-8527-66f742a22436">

Reply:
After the conversation with Eliya, he advised that I use the suggestion he gave in the issue comment I posted on the description of this PR. He gave some reasons, which are included in these screenshots below
https://github.com/upleveled/eslint-config-upleveled/pull/239/commits/317c0eb9dadda3b3db02a0858a779485e843982a

<img width="1007" alt="Screenshot 2023-05-23 at 05 57 41" src="https://github.com/upleveled/eslint-config-upleveled/assets/74430629/64c1c87e-f0ea-4293-8e38-6ca02cdc3d48">

<img width="1007" alt="Screenshot 2023-05-23 at 05 58 03" src="https://github.com/upleveled/eslint-config-upleveled/assets/74430629/e8d9a19a-89b2-4edd-8527-66f742a22436">